Resources/public/js/views/tabs/ez-locationviewlocationstabview.js

```diff
@@ -98,7 +101,40 @@ YUI.add('ez-locationviewlocationstabview', function (Y) {
          */
         _refresh: function () {
             this._fireLoadLocations();
-        }
+        },
+
+        /**
+         * Fire the `switchVisibility` event
+         *
+         * @method _fireSwitchVisibility
+         * @protected
+         * @param {EventFacade} e
+         */
+        _fireSwitchVisibility: function (e) {
```

Comment:
it does more than just firing the event now, a better name would be  `_switchVisibility`